examples/connect-app-teleop-react/src/env.ts

```diff
@@ -0,0 +1,41 @@
+export interface BuildEnvironment {
+  baseUri: string;
+  orgId: string;
+  isDev: boolean;
+  auth:
+  | { case: 'api_key'; apiKeyId: string; apiKeySecret: string }
+  | { case: 'third_party'; clientId: string };
```

Comment:
Looking at the implementation, should `auth` be optional?

```suggestion
  auth?:
  | { case: 'api_key'; apiKeyId: string; apiKeySecret: string }
  | { case: 'third_party'; clientId: string };
```

Reply:
To work with app you need some form of auth so yeah, it's necessary